suite/tests/CMakeLists.txt

```diff
@@ -553,6 +553,24 @@ function(add_exe test source)
       "${CMAKE_GENERATOR}" MATCHES "Visual Studio")
     add_dependencies(${test} ${gen_asm_tgt})
   endif ()
+  if (BUILD_TESTS_SVE)
+    target_compile_options(${test} PRIVATE
+      "-march=armv8.4-a+crypto+rcpc+sha3+sm4+sve+rng+ssbs+nodotprod")
```

Comment:
Normally we would use `CHECK_C_COMPILER_FLAG` to confirm that a flag is actually supported.

Reply:
Have added TODO here to replace with CHECK_C_COMPILER_FLAG